intellij/test/junit/saros/intellij/filesystem/IntelliJReferencePointManagerTest.java

```diff
@@ -0,0 +1,107 @@
+package saros.intellij.filesystem;
+
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.roots.ModuleRootManager;
+import com.intellij.openapi.vfs.VirtualFile;
+import org.easymock.EasyMock;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import saros.filesystem.IPath;
+import saros.filesystem.IReferencePoint;
+
+public class IntelliJReferencePointManagerTest {
+
```

Comment:
I don't know why you need all these fields. I think the majority is unused.
Only `intelliJReferencePointManager` and `referencePoint` are used.